pages/home.jsx

```diff
@@ -314,15 +314,15 @@ var HomePage = React.createClass({
             />
           </IconButtons>
         </HeroUnit>
-        <div className="row full-row mozfest-banner">
+        <div className="row full-row promo-banner">
           <div className="inner-container">
             <section>
               <Illustration
-                height={200} width={384}
-                src1x="/img/pages/home/Mozilla-Festival_Logo-Lockup_White.png" src2x="/img/pages/home/Mozilla-Festival_Logo-Lockup_White@2x.png"
+                height={190} width={190}
```

Comment:
> are we basing this off of the image itself, to make sure it's sized correctly?

yup yup

> If so, we can probably get that off of the file by using a node module for checking metadata on png/jpg

Cool! Does that module automatically add `width` and `height` attributes to the underlying `<img>` tag? (mind creating an issue for this?)